docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>here the indentation of line 2 seems a bit odd, I'd align it with the batch_size argument.</p><p> </p><p>Please set <code><code>num_threads=3</code></code></p><p>About <code>batch_size=8</code>.   Please unify to use either this in all examples of the file, or <code>batch_size=max_batch_size</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>